peps/pep-0767.rst

```diff
@@ -0,0 +1,548 @@
+PEP: 767
+Title: Annotating Read-Only Attributes
+Author: Eneg <eneg at discuss.python.org>
+Sponsor: Carl Meyer <carl@oddbird.net>
+Discussions-To: <REQUIRED: URL of current canonical discussion thread>
+Status: Draft
+Type: Standards Track
+Topic: Typing
+Created: 18-Nov-2024
+Python-Version: 3.14
+
+
+Abstract
+========
+
+:pep:`705` introduced the :external+py3.13:data:`typing.ReadOnly` type qualifier
+to allow defining read-only :class:`typing.TypedDict` items.
+
+This PEP proposes expanding the scope of ``ReadOnly`` to class and protocol
+attributes, as a single concise way to mark them read-only. Some parity changes
+are also made to :data:`typing.Final`.
```

Comment:
In the current draft the changes to `Final` are limited to clarifying that final attributes cannot be deleted, right? Maybe rephrase this to better reflect that the situation, something like
```suggestion
attributes, as a single concise way to mark them read-only. One small clarification
is also made to the specification of :data:`typing.Final`.
```

Reply:
I was planning to remove this note altogether, but this is fine too